Automattic/wp-calypso
# Privacy Policy Banner: migrate CSS to webpack, useTranslate hook

Migrates the Privacy Policy Dialog styles to webpack. The `Banner` uses just the generic banner component, with no custom styling at all.

Two tiny drive-by updates:
- extract the `shouldRender` code to a separate method.
- `useTranslate` hook in the dialog.

**How to test:**
- go to `/stats/example.blog?flags=privacy-policy/test`
- verify that the Privacy Policy banner is displayed on the stats page and that the "Learn more" dialog is styled well.
